14-01-date-input/14-1-EN-date-input.sbv

```diff
@@ -35,19 +35,19 @@ I mean "month/day/year".
 Last row dates are in Unix timestamp format.
 
 0:00:41.860,0:00:46.240
-Thats number of seconds that have elapsed since January 1, 1970 00:00 UTC
+That's number of seconds that have elapsed since January 1, 1970 00:00 UTC
 
 0:00:46.540,0:00:50.800
 It's a widely used format to represent a date as unique number.
 
 0:00:50.860,0:00:54.660
-Its used in software development and git understands it too.
+It's used in software development and Git understands it too.
 
 0:00:55.320,0:00:57.660
-As for now everything looks pretty simple, isnt't it?
+As for now everything looks pretty simple, isn't it?
 
 0:00:58.020,0:01:01.320
-It gets fun when the date is set partly.
+When date is set partly this is where the fun begins.
```

Comment:
Whats wrong with the old one?

Reply:
sorry, here i really didn't feel it. that's the only reason. 
'this is where fun begins' common phrase, think is more adapted.